Source/AnimatableItem.swift

```diff
@@ -10,6 +10,42 @@ import UIKit
 
 public protocol AnimatableItem: AnyObject {
     var position: CGPoint { get }
+    var scale: CGFloat { get }
+    var scaleX: CGFloat { get }
+    var scaleY: CGFloat { get }
+    var cgOpacity: CGFloat { get }
```

Comment:
`cgOpacity`? What does `cg` mean?

Reply:
Due to layer have property `opacity: Float` it makes invalid redeclaration with another type. So now I renamed this property as `alpha: CGFloat`, we can create issue to improve this conformance.